src/components/feature-board/content-elements/FileContentElementChips.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<div class="ma-2">
+		<v-chip class="mr-2" small>{{ fileExtension }}</v-chip>
+		<v-chip class="mr-2" small>{{ humanReadableFileSize }}</v-chip>
```

Comment:
I also found, that the `v-chips` are still having an `hover` effect. Please apply the changes suggested by @SevenWaysDP .

```suggestion
		<v-chip class="mr-2" disabled small>{{ fileExtension }}</v-chip>
		<v-chip class="mr-2" disabled small>{{ humanReadableFileSize }}</v-chip>
```

Reply:
I'm wondering where this hover effect comes from because a v-chip doesn't normally have it. What I found is that when hovering the following pseudo-element is rendered: ".theme--light.v-chip:hover::before" with style "opacity: 0.04;". Do you have any idea where that comes from?

Reply:
Unfortunately, I don't. I also found while doing some previous research (see [Colored section of Chips in vuetify docs](https://v2.vuetifyjs.com/en/components/chips/#colored) , that the "default" chip does have a hover effect. If you change the `color` of it, the hover effect disappears.
But as we are using the "default" color here, please implement the workaround kindly provided by @SevenWaysDP.